src/MessageHandler.py

```diff
@@ -0,0 +1,36 @@
+import urllib2
+from src.CastsCommandHandler import CastsCommandHandler
+from src.MessageRouter import MessageRouter
+from src.StreamsCommandHandler import StreamsCommandHandler
+
+
+class MessageHandler:
+    """
+    This class is responsible for constructing the object graph needed
+    to handle messages and to delegate to it.
+    """
```

Comment:
Yes it would be something like:

```
handlers = { ... }

def MessageHandlerBuilder(config):
    def builder():
        router = MessageRouter(
            command_handlers=_handlers,
            default_rate_limit_in_seconds=0
        )
        return router.handle_message
    return builder
```

Since the class isn't really utilizing the fact that it is a class, there isn't a particular reason of it to be one. It could just be module-level functions and data, which is simpler to reason about, but perhaps a bit odd if coming from a pure class-based language.

In either case, I still think this abstraction (Whether it be defined as a function or a class with only one public method) is a bit needless, given my [above comment](https://github.com/FAForever/fafbot/pull/22#discussion_r27793791).